bundles/org.openhab.binding.danfossairunit/src/main/resources/OH-INF/i18n/danfossairunit.properties

```diff
@@ -54,6 +54,8 @@ channel-group-type.danfossairunit.temps.channel.room_temp_calculated.description
 
 channel-type.danfossairunit.currentTime.label = Current Time
 channel-type.danfossairunit.currentTime.description = Current time reported by the air unit
+channel-type.danfossairunit.defrost.label = Defrost
+channel-type.danfossairunit.defrost.description = Defrost status. Active when low outdoor temperatures pose a risk of ice formation in the heat exchanger.
```

Comment:
Nitpit: the description has a full stop (".") in the end, the others don't.

Reply:
Nice catch! Fixed.